frontend/src/main-components/Header.js

```diff
@@ -41,21 +41,20 @@ const Header = (props) => {
    * If the device size is less than 700px wide then we will display
    * a button for our side menu instead of our logo.
    */
-  const width = window.innerWidth;
+   const width = window.innerWidth;
   if (width < 700) {
```

Comment:
This should be done with CSS. Currently, if the width changes then this won't update.

https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries